model/src/main/resources/reference.conf

```diff
@@ -15,12 +15,18 @@ ort {
 
   analyzer {
     allowDynamicVersions = true
+    enablePackageManagersByDefault = true
```

Comment:
I'm still not convinced that having this feature flag is a good idea, but I won't block it either. However, I'd like us to come up with a better name, as with `enablePackageManagersByDefault = false` it's not clear enough for *me* that it means *all* PMs are disabled by default.

It would be much easier with an *enum*, like `defaultPackageManagerState = {enabled,disabled}`, but if we'd like to stick with a *boolean*, maybe `disableAllPackageManagers` is better, as it both inverts the logic and explicitly mentions "All"?

Reply:
I'm fine with the suggestions but I also like the one I proposed.
@oss-review-toolkit/core-devs As this is a breaking configuration change it would be nice to hear another opinion as well (to the change in general, not just the option name).

Reply:
So I thought about the name again, if there is an option called `enablePackageManagersByDefault` and it is set to `false`, I don't understand why a user should not assume that this means that all package managers are disabled if not explicitly enabled.

Reply:
> I don't understand why a user should not assume that this means that all package managers are disabled if not explicitly enabled.

As mentioned before, I'm mostly missing an explicit "all".

But also, what this option should emphasize is that the default of enabling all PMs gets changed to disabling all PMs. So the emphasis is on "disable". That's why I'd prefer to also have "disable" in the name.

But like I said, I won't block this and will discard my change request; the naming doesn't really matter too much if I'm still having problems with the feature per se. I'll approve today if there's no other approval / further comments.